packages/server/src/service-worker.ts

```diff
@@ -117,7 +117,10 @@ async function refetch(request: Request): Promise<Response> {
  * Whether a given URL should be broadcast
  */
 function shouldBroadcast(url: URL): boolean {
-  return url.origin === location.origin && url.pathname.includes('/api/drive');
+  return (
+    url.origin === location.origin &&
+    (url.pathname.includes('/api/drive') || url.pathname.includes('/stdin/'))
```

Comment:
I'd use the same naming convention as for the drive access. A better name for this would probably have been `/sw-api` or `/service-worker-api`, but it's out of scope for this PR.

```suggestion
    (url.pathname.includes('/api/drive') || url.pathname.includes('/api/stdin/'))
```

Reply:
I can change this, but it will need corresponding changes in `jupyterlite/xeus`and `pyodide-kernel`.

Reply:
Done in 17a6c0a.

Reply:
Thanks! Sorry for missing this in the other PRs. I just release jupyterlite-xeus 4.0.0a7 with the changes